android/app/src/main/res/values/strings.xml

```diff
@@ -12,4 +12,47 @@
     <string name="join_existing_clan">Join existing clan</string>
     <string name="clan_name">Clan Name</string>
     <string name="add_a_clan_member">Add a clan member</string>
+    <string name="new_quest">New Quest</string>
+    <string name="select_a_quest">Select a quest</string>
+    <string name="select_date">Select Date</string>
+    <string name="select_time">Select Time</string>
+    <string name="does_not_repeat">Does not repeat</string>
+    <string name="mandatory">Mandatory</string>
+    <string name="description">Description</string>
+
+    <string-array name="quest_array">
+        <item>Select a quest</item>
```

Comment:
These should absolutely be stored server-side so we can add them dynamically without updating the app.

Reply:
@keller00 

I will leave them in the app for now
with a TODO to remove